flumine/backtest/utils.py

```diff
@@ -42,3 +42,35 @@ def __init__(
         self.size_cancelled = size_cancelled
         self.cancelled_date = cancelled_date
         self.error_code = error_code
+
+
+class SimulatedUpdateResponse:
+    def __init__(
+        self,
+        status: str = None,
+        bet_id: str = None,
```

Comment:
This response just contains status, instruction (can ignore for now) and error_code

https://github.com/liampauling/betfair/blob/master/betfairlightweight/resources/bettingresources.py#L1034